lib/diagnostic.js

```diff
@@ -6,15 +6,30 @@
 // splits it into an array of normalized words
 // (uppercase strings without punctuation)
 // and returns that array.
-const normalizeWords = function (/* */) {
-  /* your response here */
+const para = 'THIS is a TEST TEST TEST ///... SENT cannnot figure GRUNTIT WORKS IF yOu paSTE inTO nODE'
+let normalizedWords = []
+
+const normalizeWords = function (test) {
+  let words = test.split(/\s+/);
+  console.log(words)
+  for (let i = 0; i < words.length; i++){
+    const word = words[i].replace(/\W+/, '')
+    normalizedWords[i] = word.toUpperCase()
+  }
+  console.log(normalizedWords);
 }
+const testForUniqe = normalizeWords(para)
 
 // 2. Write a function that takes a string argument
 // and returns an array of unique normalized words.
-const uniqueWords = function (/* */) {
-  /* your response here */
+let uniqueWordsAsKeys = function(test) {
+  for (let i = 0; i < normalizedWords.length; i++) {
+  let word = normalizedWords[i];
+  uniqueWordsAsKeys[word] = true;
```

Comment:
This is the name of your function, and not a separate object. Be careful on naming your variables!